### PR TITLE
Cleanup node scripts

### LINKED ISSUE #3670: Add - between words in npm scripts
## Description
I'm trying to submit my first PR to this awesome project.
There were some challenges that I was/am facing
e.g.
- I can't run `npm run format` from command line on Windows since it complains of not being able to run `.sh` scripts
C:\dev\idea\nx>npm run format
```bash
C:\dev\idea\nx>npm run format

> @nrwl/nx-source@10.1.0 format C:\dev\idea\nx
> ./scripts/format.sh

'.' is not recognized as an internal or external command,
operable program or batch file.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @nrwl/nx-source@10.1.0 format: `./scripts/format.sh`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @nrwl/nx-source@10.1.0 format script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\vivzt\AppData\Roaming\npm-cache\_logs\2020-09-06T14_40_28_761Z-debug.log

```
- Running the same `format` script from my IDE (IntelliJ) results in `*.md` files changing to have `LF` line endings and show up as changed.
  - There are few ways we could resolve this issue. One option that comes to mind is adding `.gitattributes` to make this platform agnostic. I'm open to suggestions and willing to submit a PR for this based on Nx team's feedback.

----

Now about the trivial change I was trying to submit as my first PR - https://github.com/nrwl/nx/pull/3672
I'd like to suggest that the scripts 

- `checkcommit`
- `checkformat`
- `checkimports`
- `checkversions`

be renamed to:

- `check-commit`
- `check-format`
- `check-imports`
- `check-versions`

### PR DESCRIPTION
**Pull request by vivekmore**
## Related Issue(s)

Fixes #3670

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,15 +8,19 @@
...
     "commit": "git-cz",
-    "checkcommit": "node ./scripts/commit-lint.js",
+    "checkcommit": "echo 'Please use check-commit as this script is deprecated'",
+    "check-commit": "node ./scripts/commit-lint.js",
+    "checkformat": "echo 'Please use check-format as this script is deprecated'",
+    "check-format": "scripts/check-format.sh",
+    "checkimports": "echo 'Please use check-imports as this script is deprecated'",
+    "check-imports": "node ./scripts/check-imports.js",
+    "checkversions": "echo 'Please use check-versions as this script is deprecated'",
+    "check-versions": "ts-node -P ./scripts/tsconfig.scripts.json ./scripts/check-versions.ts",
```
Comment (vivekmore) [inline package.json:18]: if the said update is approved by the Nx team, please let me know if the older scripts (the ones without hyphens should be retained)

File: package.json [hunk 2]
```diff
@@ -246,7 +250,7 @@
   },
   "husky": {
     "hooks": {
-      "pre-push": "yarn checkcommit && yarn documentation && pretty-quick --check"
+      "pre-push": "yarn check-commit && yarn documentation"
```
Comment (vivekmore) [inline package.json:253]: this is definitely a temporary measure, until I receive some guidance on committing/developing on Windows OS

**Comment by vivekmore** [comment]
@vsavkin - thanks I'll look into working with WSL (or better yet making things work with Windows)

But do you like the change (rename of node scripts) I'm proposing here:
https://github.com/nrwl/nx/pull/3672/commits/a874ac2866e790dd5d4300154d851aa39c1af65c

File: package.json [hunk 3]
```diff
@@ -8,15 +8,19 @@
...
     "commit": "git-cz",
-    "checkcommit": "node ./scripts/commit-lint.js",
+    "checkcommit": "echo 'Please use check-commit as this script is deprecated'",
+    "check-commit": "node ./scripts/commit-lint.js",
+    "checkformat": "echo 'Please use check-format as this script is deprecated'",
+    "check-format": "./scripts/check-format.sh",
+    "checkimports": "echo 'Please use check-imports as this script is deprecated'",
+    "check-imports": "node ./scripts/check-imports.js",
+    "checkversions": "echo 'Please use check-versions as this script is deprecated'",
+    "check-versions": "ts-node -P ./scripts/tsconfig.scripts.json ./scripts/check-versions.ts",
```
Comment (vivekmore) [inline package.json:18]: Please let me know if the older scripts (the ones without hyphens should be retained)

**Comment by vivekmore** [comment]
@vsavkin - I have followed the steps to setup dev environment with WSL2 and updated the PR.
Please let me know if it looks good to you.

File: package.json [hunk 4]
```diff
@@ -8,15 +8,19 @@
   "scripts": {
     "build": "./scripts/package.sh --local",
     "commit": "git-cz",
-    "checkcommit": "node ./scripts/commit-lint.js",
+    "checkcommit": "echo 'Please use check-commit as this script is deprecated'",
```
Comment (FrozenPandaz) [inline package.json:11]: Please remove the old ones.

File: package.json (hunk 3 above)
Comment (FrozenPandaz) [reply package.json:18]: We don't need to retain them.

**Review by FrozenPandaz — CHANGES_REQUESTED** [review]
Thank you for these changes! Please remove the old scripts.

**Comment by vivekmore** [comment]
> Thank you for these changes! Please remove the old scripts.

done

**Review by FrozenPandaz — APPROVED** [review]

**Pull request merged**